models/review.go

```diff
@@ -65,7 +70,8 @@ type Review struct {
 	// CodeComments are the initial code comments of the review
 	CodeComments CodeComments `xorm:"-"`
 
-	Comments []*Comment `xorm:"-"`
+	Comments     []*Comment `xorm:"-"`
```

Comment:
sorry, I accidentally added a few Spaces, I wll recover it in next commit.
```suggestion
	Comments []*Comment `xorm:"-"`
```